language/tools/move-mv-llvm-compiler/src/stackless/translate.rs

```diff
@@ -121,6 +121,7 @@ impl<'up> GlobalContext<'up> {
         &'this self,
         id: mm::ModuleId,
         dot_info: &'this String,
+        log_level: String,
```

Comment:
It is orthogonal to this patch, but now that we are adding more command line options, it isn't going to be scalable (or at least desirable) to keep adding each one to the contexts. It probably makes sense now to make `main.rs:Args` publicly/globally available.